README.md

```diff
@@ -121,8 +121,11 @@ You can also join us on:
 
 - Freenode IRC - ``#ansible-aws`` Freenode channel
 
-See the [Ansible Community Guide](https://docs.ansible.com/ansible/latest/community/index.html) for details on contributing to Ansible.
+### More information about contributing
 
+- [Ansible Community Guide](https://docs.ansible.com/ansible/latest/community/index.html) - Details on contributing to Ansible
```

Comment:
```suggestion
- [Ansible Community Guide](https://docs.ansible.com/ansible/latest/community/index.html) - Details on contributing to Ansible
- [Contributing to Collections](https://docs.ansible.com/ansible/devel/dev_guide/developing_collections.html#contributing-to-collections) - How to check out collection git repositories correctly
```
While this is also described in @jillr's blogpost (section Testing Locally), I don't think it hurts to repeat it, since this seems to be a common cause of frustration.